tools/please_pex/pex/pex_main.py

```diff
@@ -191,7 +191,7 @@ def find_spec(self, name, path, target=None):
 
     def create_module(self, spec):
         """Actually load a module that we said we'd handle in find_module."""
-        module = import_module(spec.name[len(self.prefix):])
+        module = import_module(spec.name.removeprefix(self.prefix))
```

Comment:
drive-by here, this was introduced in 3.9 which is the minimum version we support.